lib/src/transform/bake_orientation.dart

```diff
@@ -13,8 +13,14 @@ Image bakeOrientation(Image image) {
     return bakedImage;
   }
 
-  // Clear the exif data.
-  bakedImage.exif.data.remove(ExifData.ORIENTATION);
+  // Copy all exif data except for orientation
+  bakedImage.exif = ExifData();
```

Comment:
Have the same issue, oddly enough the logic you apply here should be the same as `.remove()` 🤔

Reply:
> Have the same issue, oddly enough the logic you apply here should be the same as `.remove()` 🤔

Almost. The difference is the new ExifData() object also clears RawData. This solution is equivalent to using .remove AND clearing rawData().

Not the ideal solution, but better than what is in the master branch right now. I will try to properly parse and modify rawData when I have time, hopefully this month.